src/popup.js

```diff
@@ -519,6 +519,35 @@ window.addEventListener('load', function(evt) {
 
     window.close();
   });
+  
+  $(".wcmmode").click(function(e) {
+    e.preventDefault();
+    
+    var $this = $(this),
+        url = getUrlWithUpdatedQueryString(pageDetails.location, null),
```

Comment:
Probably, I am very new to angular, and haven't had the time to go in depth with your plugin code, I just wanted to make it use cookies rather than query param.